crates/sui-indexer/migrations/2023-08-19-044023_objects/up.sql

```diff
@@ -10,6 +10,10 @@ CREATE TABLE objects (
     owner_id                    bytea,
     -- Object type
     object_type                 text,
+    -- Components of the StructTag: package, module, name
+    object_type_package         bytea,
+    object_type_module          text,
+    object_type_name            text,
```

Comment:
also add index for the newly added columns?

Reply:
Added.